go.mod

```diff
@@ -87,7 +87,7 @@ require (
 	github.com/smartcontractkit/chainlink-framework/multinode v0.0.0-20241220173418-09e17ddbeb20
 	github.com/smartcontractkit/chainlink-protos/orchestrator v0.4.0
 	github.com/smartcontractkit/chainlink-solana v1.1.1-0.20250110142550-e2a9566d39f3
-	github.com/smartcontractkit/chainlink-starknet/relayer v0.1.1-0.20241202202529-2033490e77b8
+	github.com/smartcontractkit/chainlink-starknet/relayer v0.1.1-0.20250115160455-c8da77327e9c
```

Comment:
we'll need to update this to the commit hash here right? https://github.com/smartcontractkit/chainlink-starknet/pull/555 (starts with '6211...')

Reply:
It will need the final version to merge, but we are stuck in the chicken-egg scenario to get the test to pass from the other PR.